cloud/configs/nginx.conf

```diff
@@ -5,6 +5,10 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 
+# WARNING: Changes to this file will not be reflected until the
+# running nginx instance is explicity reloaded `nginx -s reload` or
```

Comment:
Thanks for looking into that Matt. I think having a way to non-disruptively restart nginx is very useful, in part because it will save other people from having to figure out how to do it manually. It could be either part of the postinst script, which is the approach I took and then abandoned, or it could be a step run by fab if the deploy is successful. I'd probably lean towards the former,  since then the config reload will take place if a deployment is done in some way other than using fab, e.g.., thru the AWS CodeDeploy console.